services/version-api/helpers/getApplicationUpdateStatus.js

```diff
@@ -0,0 +1,34 @@
+import smCompare from 'semver-compare';
```

Comment:
`smCompare` seems like an odd abbreviation. `compareSemver`? `compareVersion`?

Reply:
Will change it to`semverCompare`